29 make the api mock server

taniguchiiqqq opened the pull request:
## Overviews of implementation

added docker-compose.yaml and Dockerfile to run the API mock server based on Nishiki-document's OAS

## Review points

Ensure that the mock server can start by `npm run mock` command, and you can get a response when you access [http://localhost:8080/users/aaa](http://localhost:8080/users/aaa)

hitohata reviewed (approved):
@taniguchiiqqq 
Thank you for implementation.
I think you need to write commands on the README. And it seems good if you add the 'docker-compose down' command in the scripts.

taniguchiiqqq commented:
@hitohata 
I appreciate your review! I updated Readme and removed the detach option from the docker-compose command because other commands work in the foreground.

hitohata commented:
@taniguchiiqqq 
Thank you. sounds good. I've already approved this PR, so feel free to merge it.
